_sass/_va.scss

```diff
@@ -332,7 +332,7 @@ a:hover {text-decoration: underline; background: rgba(0,0,0,.1);}
 }
 
 html.no-touch .banner {
-  background: url(../img/design/banner.jpg) 50% 60% no-repeat;
+//  background: url(../img/design/banner.jpg) 50% 60% no-repeat;
```

Comment:
I don't see this style even being applied anywhere (can't find an instance of the class `banner` being applied).